Amazon.js

```diff
@@ -215,9 +216,10 @@ function scrape(doc, url) {
 	) {
 		baseNode = baseNode.parentElement;
 	}
-	
+
+	var authors, name, role, invertName;
 	if (baseNode) {
-		var authors = ZU.xpath(baseNode, './/span[@id="artistBlurb"]/a');
+		authors = ZU.xpath(baseNode, './/span[@id="artistBlurb"]/a');
```

Comment:
Just to note, unless a variable is used outside a block, we prefer to use `let` inside the block rather than `var` outside — lets you easily know that a variable's scope is limited to the current block. (We also try to avoid multi-variable initialization, since it makes for worse diffs.)

Reply:
got it, thanks!